module/x/gravity/types/ethereum_signer.go

```diff
@@ -12,18 +12,18 @@ const (
 	signaturePrefix = "\x19Ethereum Signed Message:\n32"
 )
 
-// NewEthereumSignature creates a new signuature over a given byte array
-func NewEthereumSignature(hash []byte, privateKey *ecdsa.PrivateKey) ([]byte, error) {
+// NewEVMSignature creates a new signature over a given byte array
+func NewEVMSignature(hash []byte, privateKey *ecdsa.PrivateKey) ([]byte, error) {
 	if privateKey == nil {
 		return nil, sdkerrors.Wrap(ErrInvalid, "did not pass in private key")
 	}
 	protectedHash := crypto.Keccak256Hash(append([]byte(signaturePrefix), hash...))
 	return crypto.Sign(protectedHash.Bytes(), privateKey)
 }
 
-// ValidateEthereumSignature takes a message, an associated signature and public key and
+// ValidateEVMSignature takes a message, an associated signature and public key and
 // returns an error if the signature isn't valid
-func ValidateEthereumSignature(hash []byte, signature []byte, ethAddress common.Address) error {
+func ValidateEVMSignature(hash []byte, signature []byte, ethAddress common.Address) error {
```

Comment:
Can we just rename these functions in-place, or will backwards compatibility require alternate versions? Does the `signaturePrefix` apply the same to non-Ethereum chains?